crates/krator/src/runtime.rs

```diff
@@ -155,8 +155,8 @@ impl<O: Operator> OperatorRuntime<O> {
                         }
                         match manifest_tx.send(manifest) {
                             Ok(()) => (),
-                            Err(e) => {
-                                warn!("Unable to broadcast manifest update: {:?}", e);
+                            Err(_) => {
+                                debug!("Manifest receiver hung up, exiting.");
```

Comment:
It turned out that this never happened because all of our receivers were stuck. Now that they exit properly, this happens, but should not be considered an error. I updated the three places where this type of behavior was logged.